### PR TITLE
Release updates

### PR DESCRIPTION
**Pull request by dansiegel**
﻿### Description of Change ###

Updating Dependencies & bumping version for public preview

### Bugs Fixed ###

n/a

### API Changes ###

n/a

### Behavioral Changes ###

n/a

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [X] Rebased on top of master at time of PR
- [X] Changes adhere to coding standard

### CODE REVIEW

File: Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <Version Condition=" '$(PRISM_RELEASE)' != '' ">$(PRISM_RELEASE)</Version>
     <Version Condition=" '$(PRISM_AUTOFAC_WINDOWS_VERSION)' != '' ">$(PRISM_AUTOFAC_WINDOWS_VERSION)</Version>
     <!-- Update this version when incrementing Library Versions i.e. 7.1.0, 7.2.0 -->
-    <VersionPrefix Condition=" '$(PRISM_RELEASE)' == '' And '$(PRISM_AUTOFAC_WINDOWS_VERSION)' == '' ">7.0.0</VersionPrefix>
+    <VersionPrefix Condition=" '$(PRISM_RELEASE)' == '' And '$(PRISM_AUTOFAC_WINDOWS_VERSION)' == '' ">7.1.0</VersionPrefix>
```
Comment (bartlannoeye) [inline Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj]: bumping windows as well?

Comment (bartlannoeye) [inline Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj]: bumping windows as well?

Comment (brianlagunas) [reply Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj]: It's just a formality. UWP as we know it is going away.

Comment (dansiegel) [reply Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj]: @totht91 we are currently planning on replacing our current Prism for UWP libraries with a merge of Prism and Template10

Comment (totht91) [reply Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj]: Could you please give us some information about the merge? How can the community help you with this?

**Pull request merged**